File: python programming/nestedforloop1.py
n=int(input("enter no of term:-"))
sum=0
for i in range(1,n+1):
 fact=1
 for j in range(1,i+1):
  fact=fact*j
 print(f'{i}!',end='')
 if i<n:
  print(' + ',end='')
 sum+=fact
print('\nsum of seriese=',sum)

File: CppCompiler/nestedforloop1.cpp
#include<iostream>
using namespace std;

int main()
{
 int i,n,j,fact=1,sum=0;
cout<<"enter no of term:-";
cin>>n;
for(i=1;i<=n;i++)
{
cout<<i<<"!";
if(i<n)
{
cout<<"+";
}
for(j=1;j<=i;j++)
{
fact=fact*j;
}
sum=sum+fact;
fact=1;
}
cout<<"="<<sum;
return 0;
}

